wizeline/wizeline-bot-webchat
# Adding gzip settings for static files

### What this PR does?
Adds nginx gzip compression for static files

### How to test it?
Run the application with nginx 
```docker build -t wizeline-webchat .
docker run -p 80:80 
```
Access the app in http://localhost:80
Open your inspect console / network
Check the compression is enabled

![screen shot 2017-02-02 at 20 54 31](https://cloud.githubusercontent.com/assets/748051/22578207/9abe04d0-e98c-11e6-8681-b7cd5eeb1bd2.png)
![screen shot 2017-02-02 at 20 54 04](https://cloud.githubusercontent.com/assets/748051/22578208/9abe947c-e98c-11e6-9e0a-aee5371329e0.png)
![screen shot 2017-02-02 at 20 52 30](https://cloud.githubusercontent.com/assets/748051/22578209/9abe9ab2-e98c-11e6-9ec2-c32725540b65.png)

http://wizebot-webchat-houston-develop.wize.mx/views/?s=asdasdasdasdasd
### Any background?

For faster transportation over the network

### Comments

Don't break it
